hummingbird/ml/supported.py

```diff
@@ -34,11 +35,15 @@ def _build_sklearn_operator_list():
     Put all suported Sklearn operators on a list.
     """
     if sklearn_installed():
+        # Enable experimental to import HistGradientBoostingClassifier
+        from sklearn.experimental import enable_hist_gradient_boosting
```

Comment:
it's because the HistGBDT estimators are still experimental and are not subject to the 2-versions deprecation cycles, so we requires users to explicitly say "I want to use this feature anyway"

When the estimators become stable, this will have no effect